File: Renderable.cpp
#include "Renderable.h"
 
void Renderable::render(){
    for(int i=0; i<Cell::getSizex(); i++){
        for(int j=0l; j<Cell::getSizey(); j++){
            cout << Cell::getElement(i,j);
        }
        cout << endl;
    }
}

File: java/Board/Renderable.java
class Renderable extends Cell{
    public void render(){
        for(int i=0; i<super.getSizeX(); i++){
            System.out.print("_");
            for(int j=0; j<super.getSizeY(); j++) {
                System.out.print("__");
            }
            System.out.println(); 
            System.out.print("|");
            for(int j=0; j<super.getSizeY(); j++){
                if (super.getElement(i,j) != ' '){
                    System.out.print(super.getElement(i,j) + "|");
                } else if (super.getRumput(i,j) != ' '){
                    System.out.print(super.getRumput(i,j) + "|");
                } else {
                    System.out.print(super.getTypeLand(i,j) + "|");
                }
            }
            System.out.println();
        }
    }
}